core/templates/dev/head/components/subtitled_html_editor_directive.html

```diff
@@ -0,0 +1,5 @@
+<audio-translations-editor component-name="<[COMPONENT_NAME_HTML]>"
```

Comment:
This is wrong, components should vary because this is how we distinguish filenames. Hints should get COMPONENT_NAME_HTML, etc.